Resources/Prototypes/_NF/Events/events.yml

```diff
@@ -57,3 +57,18 @@
   - type: BluespaceErrorRule
     gridPath: /Maps/Bluespace/vaultsmall.yml
     rewardFactor: 1
+
+- type: entity
+  id: BluespaceSyndicateCrate
+  parent: BaseGameRule
+  noSpawn: true
+  components:
+  - type: StationEvent
+    startAudio:
+      path: /Audio/Announcements/attention.ogg
+    weight: 1
+    startDelay: 30
+    duration: 35
+    earliestStart: 90
+    minimumPlayers: 20
```

Comment:
maybe more like 25-30 players, we need to make sure there's likely at least some kind of sec force around.

Reply:
eh lets just role with it why not